### PR TITLE
Does my coding partner follow JS best practices?

### PR DESCRIPTION
**Pull request by Mutai-Gilbert**
This pull request is created for a code review between coding partners to see it follows best coding practices.

Checking the quality of the code. The following will be checked:

1. JavaScript best practices.
2. DRY, KISS, and YAGNI.
3. HTML & CSS best practices

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1 +1,162 @@
...
+      this.firstDiv.appendChild(this.contentSpan);
+      this.li.appendChild(this.firstDiv);
+      this.secondDiv.appendChild(this.secondElement);
+      this.secondDiv.appendChild(this.trash);
+      this.secondDiv.appendChild(this.edit);
+      this.li.appendChild(this.secondDiv);
+      this.theTodos.appendChild(this.li);
+    });
+    localStorage.setItem('tasks', JSON.stringify(this.tasks));
+  }
```
Comment (stephenkati) [inline src/index.js:147]: Make use of JavaScript ES6, employ use of arrow function functions to make your code easy to read and maintain

File: src/index.js [hunk 2]
```diff
@@ -1 +1,162 @@
...
+
+class TodoList {
+  constructor() {
+    this.tasks = JSON.parse(localStorage.getItem('tasks')) || [];
+    this.theTodos = document.querySelector('.the-todos');
+    this.displayTasks();
+  }
+
+  removeTask(taskIndex, liElement) {
+    // remove the task from the DOM
```
Comment (stephenkati) [inline src/index.js:12]: Remove comments, your code should be self-explanatory hence no need to add comments

**Review by stephenkati — CHANGES_REQUESTED** [review]
**Hi @Mutai-Gilbert,**
#
**Highlights!**
Good code organization and working app :heavy_check_mark:
Descriptive README:heavy_check_mark:
Linters :heavy_check_mark: 
DRY code :heavy_check_mark:
#
Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! :tada:

### **Status: Approved :heavy_check_mark:**
Cheers, and Happy coding!:clap::clap::clap:
#
**Optional suggestions**
Every comment with the **[OPTIONAL]** prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: src/index.js (hunk 1 above)
Comment (Mutai-Gilbert) [reply src/index.js]: Thank you

File: src/index.js (hunk 2 above)
Comment (Mutai-Gilbert) [reply src/index.js]: Done

File: src/index.js (hunk 1 above)
Comment (stephenkati) [reply src/index.js]: You are welcome

**Pull request merged**